wcivf/settings/base.py

```diff
@@ -279,9 +279,9 @@ def get_ec2_ip():
 # DC Base Theme settings
 SITE_TITLE = "Who Can I Vote For?"
 
-AKISMET_API_KEY = os.environ.get("AKISMET_API_KEY")
+PYTHON_AKISMET_API_KEY = os.environ.get("PYTHON_AKISMET_API_KEY")
```

Comment:
If you want to change this environment viable, you'll need to change it in the parameter store. 

However, I don't really see the need to change the var name here. The content is the same, and for me `AKISMET_API_KEY` is better than `PYTHON_AKISMET_API_KEY`...in the context of a python project the prefix feels redundant to me.

Reply:
I changed this on the advice on the package installation instructions https://akismet.readthedocs.io/en/latest/sync_client.html but happy now that I'm setting the config variable, I can just revert and do: 

`config = akismet.Config(
            key=settings.PYTHON_AKISMET_API_KEY,
            url=settings.PYTHON_AKISMET_BLOG_URL,
        )`